### YamlMime:ManagedReference
items:
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  id: RuntimeBinderException
  children:
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RuntimeBinderException
  nameWithType: RuntimeBinderException
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  type: Class
  summary: "Akkor következik be, a C# futásidejű kötő dinamikus kötési feldolgozásakor hiba jelöli."
  remarks: "Az ilyen kivételek Miben különböznek a <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>abban a RuntimeBinderException jelöli abban az értelemben, a szokásos fordítási hiba történt a kötés nem sikerült, mivel <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>maga futásidejű kötő üzemzavar jelöli.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException> </xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>"
  syntax:
    content: 'public class RuntimeBinderException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  id: '#ctor'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException()
  nameWithType: RuntimeBinderException.RuntimeBinderException()
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException()
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Egy új példányát inicializálja a <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> osztály."
  remarks: "Ez a konstruktor inicializálja a <xref:System.Exception.Message%2A>tulajdonság egy rendszer által megadott üzenetet, amely leírja a hibát, például a &quot;DefaultMessageDisplayedByParameterlessCtorWriterMustSupply&quot; ezt az üzenetet az új példány veszi figyelembe a rendszer aktuális kulturális környezetének.</xref:System.Exception.Message%2A>       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.InsufficientMemoryException>.</xref:System.InsufficientMemoryException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Az üzenet honosított hibakarakterlánc. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException ();
    parameters: []
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(String)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Egy új példányát inicializálja a <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> osztály, amely rendelkezik a megadott hibaüzenet jelenik meg."
  remarks: "Az alábbi táblázat a kezdeti tulajdonságértékek <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A hiba üzenet mezőben megadott karakterlánc `message`. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel leíró üzenet. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(SerializationInfo,StreamingContext)
  nameWithType: RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Egy új példányát inicializálja a <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> osztály, amely rendelkezik a szerializált adatok."
  remarks: "Ez a konstruktor nevezik meglévő kölcsönökre az adatfolyam továbbított kivétel objektum deszerializálása közben."
  syntax:
    content: protected RuntimeBinderException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Az objektum, amely tárolja a kivétel lépett fel az éppen szerializált objektum adatait."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A cél- és környezeti információt."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(String,Exception)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String,Exception)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String,Exception)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Egy új példányát inicializálja a <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> osztály, amely rendelkezik a megadott hibaüzenet és a belső kivételben találhatók okozza-e ezt a kivételt, mutató hivatkozás."
  remarks: "Kivételt vált ki, mert egy előző kivétel közvetlen eredménye tartalmaznia kell egy hivatkozást az előző kivétel az a <xref:System.Exception.InnerException%2A>tulajdonság.</xref:System.Exception.InnerException%2A> A <xref:System.Exception.InnerException%2A>tulajdonság a konstruktornak átadott ugyanazt az értéket adja vissza vagy `null` Ha a <xref:System.Exception.InnerException%2A>tulajdonság nem adja meg a belső kivétel értékét a konstruktor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A hiba üzenet mezőben megadott karakterlánc `message`. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A hibaüzenet, amely ismerteti a kivétel az az oka."
    - id: innerException
      type: System.Exception
      description: "A kivétel okozza-e az adott kivételt, vagy null hivatkozás. Ha meg van adva a nem a belső kivételben találhatók."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException()
  nameWithType: RuntimeBinderException.RuntimeBinderException()
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException()
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(String)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(SerializationInfo,StreamingContext)
  nameWithType: RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(String,Exception)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String,Exception)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String,Exception)
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException
  nameWithType: RuntimeBinderException.RuntimeBinderException
